packages/amplify-e2e-tests/src/__tests__/layer.test.ts

```diff
@@ -62,22 +63,53 @@ describe('amplify add lambda layer', () => {
 
   it('init a project and add/update simple layer and push', async () => {
     const [shortId] = uuid().split('-');
-    const settings = {
+    const settingsAdd = {
+      runtimes: ['nodejs'],
       layerName: `testlayer${shortId}`,
-      versionChanged: true,
     };
     await initJSProjectWithProfile(projRoot, {});
-    await addLayer(projRoot, settings);
-    await updateLayer(projRoot, settings);
+    await addLayer(projRoot, settingsAdd);
+    const settingsUpdate = {
+      runtimes: ['java'],
+      layerName: `testlayer${shortId}`,
```

Comment:
Let's add a const above so that the different settings objects are in sync

```js
   const [shortId] = uuid().split('-');
   const layerName = `testlayer${shortId}`;
   const settingsAdd = {
     layerName,
     runtimes: ['nodejs'],
     versionChanged: true,
   };
```

Reply:
Same thing in the next test.

Reply:
done thanks!!